pennylane/qnode.py

```diff
@@ -703,7 +703,7 @@ def to_tf(self, dtype=None):
         # pylint: disable=import-outside-toplevel

```

Comment:
On line 669 (not sure if we'd want it to be done in this particular PR):
>        # todo: remove 'kwargs' when tape mode is default.
>        # currently it only exists to match the signature of non-tape mode draw.

Reply:
Very good eye!

Reply:
Done